### ISSUE TITLE
[Feature Request] Add Open with... RetroArch

### ISSUE THREAD

**Issue by hyarsan** [libretro/RetroArch#5614]
here's something that would be useful...

Add Open with... RetroArch to the Windows version of RetroArch!
Since we have an installer now, it can't be that impossible? But wait...

How would cores work? Nevermind.... it would be convienient though!

**Comment by RobLoach** [comment]
Duplicate of https://github.com/libretro/RetroArch/issues/2634

**Comment by hyarsan** [comment]
Not really, I am talking about the Windows menu option "Open with..." not the command line. Although #2634 would help! =)

![this_is_no_duplicate](https://user-images.githubusercontent.com/32988993/32118104-d64c37ba-bb50-11e7-9c73-01b4257bfe70.png)
